content/docs/configuration/acme/dns01/route53.md

```diff
@@ -138,251 +490,39 @@ And the following trust relationship (Add AWS `Service`s as needed):
 }
 ```
 
-## Creating an Issuer (or `ClusterIssuer`)
+## Region
```

Comment:
In https://github.com/cert-manager/website/issues/56 @stephen-dexda and @TBBle showed that the `region` field should be optional. We made it optional inhttps://github.com/cert-manager/cert-manager/pull/7287 and updated the API docs there.
Here I have added the same background information to the the website documentation.
 * https://deploy-preview-1555--cert-manager.netlify.app/docs/configuration/acme/dns01/route53/#region 

@stephen-dexda and @TBBle: Please give feedback if you have time.

Reply:
It's been a while since I looked at this stuff (so EKS Pod Identities is new to me (and very neat!)), so I'm not in a position to validate what's written here beyond the linked documentation.

With that qualification, the Region section looks fine to me.